extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -56,7 +56,7 @@ public static function onOasisSkinAssetGroups( &$jsAssets ) {
 		if ( self::isCorrectPageType() ) {
 			$jsAssets[] = 'recirculation_js';
 
-			if ( self::canShowDiscussions() ) {
+			if ( self::canShowDiscussions( $wgCityId ) ) {
```

Comment:
The use cases are a little bit different as they are for different experiments, however the check could probably be removed from the `RecirculationController.php` file as the route that uses it should never get called without including the `recirculation_discussions_js` file referenced here. The Recirculation controller is getting an update in the near future and will probably be deprecated in favor of just using the RecirculationAPIController so I'll take a look at consolidating it then.